test/DFArena.test.ts

```diff
@@ -196,37 +205,30 @@ describe('Arena Functions', function () {
       planets.map((p) => {
         const planetArgs = {
           location: p.id,
+          x: Math.floor(Math.random() * 100),
```

Comment:
do we not want these to be determined? not sure we want to allow randomness in our testing suite